src/psyclone/tests/alggen_test.py

```diff
@@ -411,25 +419,31 @@ def test_single_invoke_dynamo0p1():
         specified in an invoke call for the dynamo0.1 API. We use the
         generate function as parse and PSyFactory need to be called before
         AlgGen so it is simpler to use this. '''
+    Config.get().api = "dynamo0.1"
```

Comment:
As discussed elsewhere: it turns out in nearly all cases setting the api is not necessary anymore (since it is actually only used in case of errors), so while I had started using monkeypatch (and finally started to understand what it does ;) ), I could remove all those assignments. It turns out that the error testing is done in functions that set the right API in a fixture.

Reply:
I'm a bit confused here - I see that you've changed the test to set the API using monkeypatch BUT it's being set to 'gocean1.0' (lines 422 and 438). I guess that the errors that those tests are provoking happen before the kernel meta-data is parsed so we get away with it?

Reply:
Having got through to the end of all the comments I think I finally understand what's going on :-) Are this line and the one I comment on below just hangovers that should be removed?

Reply:
Yes, left over ... and I am wildly cursing here: I grep'ed to find all settings of the api that I had added to the tests, but used:
```
grep "Config.get().api" *py
```
but the monkeypatched lines were not detected by the grep :( Note to self: `grep "Config.get().*api"` :)

Reply:
:-)